src/utility.c

```diff
@@ -471,3 +471,171 @@ int pathcmp(const char *a, const char *b) {
     free(myB);
     return ret;
 }
+
+int is_json_array(const char *value) {
+  if (value[0]=='[' && value[1]=='u' && value[2]=='\''){
```

Comment:
i don't understand this.  there are lots of valid json arrays that don't start with "[u'", this seems very case-specific

Reply:
we can tweak it as needed but this is what the image manager generates and that is mostly what I care about at this point.